docs/spark/tutorials/get-started-windows-instructions.md

```diff
@@ -0,0 +1,108 @@
+- Download and install **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release:
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (for example, `c:\bin\Microsoft.Spark.Worker\`).
+
+ > [!IMPORTANT]
+ >
+ > Create a [new environment variable](https://www.java.com/en/download/help/path.xml) `DOTNET_WORKER_DIR` and set it to the directory where you downloaded and extracted the Microsoft.Spark.Worker (for example, *c:\bin\Microsoft.Spark.Worker*).
+
+For detailed instructions, you can see [Building .NET for Apache Spark from Source on Windows](../how-to-guides/windows-instructions.md).
+
+## Authoring a .NET for Apache Spark App
+
+1. Open Visual Studio > **Create New Project** > **Console App (.NET Core)** > Name: `HelloSpark`.
+
+   For detailed instructions, you can see [How to create the app in Visual Studio](../../core/tutorials/with-visual-studio.md).
+
+1. Install `Microsoft.Spark` Nuget package into the solution from the [spark nuget.org feed](https://www.nuget.org/profiles/spark) - see [Ways to install Nuget Package](https://docs.microsoft.com/nuget/consume-packages/ways-to-install-a-package)
+1. Write the following code into *Program.cs*:
+
+    ```csharp
+    using Microsoft.Spark.Sql;
+
+    namespace HelloSpark
+    {
+        class Program
+        {
+            static void Main(string[] args)
+            {
+                SparkSession spark = SparkSession.Builder().GetOrCreate();
+                DataFrame df = spark.Read().Json("people.json");
+                df.Show();
+            }
+        }
+    }
+    ```
+
+1. Build the solution
+
+## Running your .NET for Apache Spark App
+
+1. Open your terminal and navigate into your app folder:
+
+    ```
+    cd <your-app-output-directory>
+    ```
+    
+1. Create *people.json* with the following content:
+
+    ```json
+    {"name":"Michael"}
+    {"name":"Andy", "age":30}
+    {"name":"Justin", "age":19}
+    ```
+    
+1. Run your app
+
+    ```bash
+    spark-submit \
+    --class org.apache.spark.deploy.dotnet.DotnetRunner \
+    --master local \
+    microsoft-spark-2.4.x-<version>.jar \
+    dotnet HelloSpark.dll
+    ```
+    
+    > [!NOTE]
+    > This command assumes you have downloaded Apache Spark and added it to your PATH environment variable to be able to use `spark-submit`. Otherwise, you'd have to use the full path (for example, *c:\bin\apache-spark\bin\spark-submit*). For detailed instructions, see [Building .NET for Apache Spark from Source on Windows](../building/windows-instructions.md).
```

Comment:
Do we want this warning to be below the command?  I wonder if a reader might not see it.

Reply:
With the [!NOTE] rending, I think it's kind obvious. Please refer to warning session [here](https://docs.microsoft.com/en-us/dotnet/spark/how-to-guides/deploy-worker-udf-binaries#parameter-options).